### DISCUSSION TITLE
Way to run node

### DISCUSSION THREAD

**Discussion by peix2** [capsule-corp-ternoa/chain#55] [Q&A]
Hi Team,

I went through white paper and some additional information and I like the idea.
What is the option of running node? I am only able to find info about development node. 
I might be missing something so apologies in advance ;-)
Cheers!
Px2

**Comment by ETeissonniere** [comment]
Hey @peix2, for now you can merely sync the chaos net. You should be able to use `cargo run --release -- --chain chaos --wasm-execution compiled` for this
